rule-packs/mitre-attck-initial-access-attack-paths.json

```diff
@@ -0,0 +1,326 @@
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND aws_iam_user WITH mfaEnabled!=true",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "initial-access-external-remote-services-use-mfa-azure",
+      "description": "M1035 - Limit Access to Resources Over Network - Use strong two-factor or multi-factor authentication for remote service accounts to mitigate an adversary's ability to leverage stolen credentials, but be aware of Multi-Factor Authentication Interception techniques for some two-factor authentication implementations in azure.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND azure_user WITH mfaEnabled!=true",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "initial-access-external-remote-services-use-mfa-google-cloud",
+      "description": "M1035 - Limit Access to Resources Over Network - Use strong two-factor or multi-factor authentication for remote service accounts to mitigate an adversary's ability to leverage stolen credentials, but be aware of Multi-Factor Authentication Interception techniques for some two-factor authentication implementations in google cloud.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND google_user WITH mfaEnabled!=true",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "initial-access-supply-chain-compromise-boot-integrity",
+      "description": "M1046 - Boot Integrity - Use secure methods to boot a system and verify the integrity of the operating system and loading mechanisms.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND google_compute_instance WITH secureBootEnabled != true",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "initial-access-supply-chain-compromise-limit-software-installation",
+      "description": "M1033 - Limit Software Installation - Where possible, consider requiring developers to pull from internal repositories containing verified and approved packages rather than from external ones.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND github_repo WITH mergeCommitAllowed = true AND visibility = 'PUBLIC'",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "HIGH"
+    },
+    {
+      "name": "initial-access-supply-chain-compromise-vulnerability-scanning-sources",
+      "description": "M1016 - Vulnerability Scanning - Continuous monitoring of vulnerability sources and the use of automatic and manual code review tools should also be implemented as well.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND github_environment THAT HAS << github_repo THAT !HAS FINDING",
```

Comment:
FIND github_environment THAT HAS << github_repo THAT !HAS **Finding**

What is this query trying to say? Repos without findings?

Reply:
Basically an informational query looking for github environments that have repos.. but DON'T have monitoring set up. Continuous monitoring of vulnerability sources.

Reply:
ok gotcha, lets do this query in that case?

FIND CodeRepo THAT !SCANS << *

Reply:
ack